packages/client/lib/sync/fullsync.ts

```diff
@@ -17,9 +22,12 @@ export class FullSynchronizer extends Synchronizer {
 
   public txPool: TxPool
 
+  private blocksKnownByPeer: Map<string, sentBlock[]>
```

Comment:
now that we are on the latest TS version, is it possible to annotate `string`? Just so it's easier to read/understand/use. If not maybe we can do here `type PeerId = string` so it can be written as `Map<PeerId, sentBlock[]>`, that seems a lot more friendly to me.

Reply:
might this also be more clear if named `newBlocksKnownByPeer` ?

Reply:
> now that we are on the latest TS version, is it possible to annotate string

A bit - a tiny bit 😋 - similar direction here: this is very analogue to problems/solutions in the tx pool implementation. My solution there were these types like `UnprefixedHash` and the like. We should just generally stay consistent here, no matter what we do.

Reply:
Looks like we can't add an additional type annotation to the type (at least not based on my small experimentation) so I went with the type alias suggested above (`peerId = string`).